src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/AndroidBrokerProxy.cs

```diff
@@ -241,6 +243,19 @@ public Intent GetIntentForBrokerActivity(AuthenticationRequest request, Activity
             return intent;
         }
 
+        private Handler GetPreferredLooper(Activity callerActivity)
+        {
```

Comment:
not sure what's involved in calling MyLooper().  should you cache the result in a var and then do the comparisons so you don't call the method 3 times?